string tagId
string objectName
float64[] objectPose